operator/Dockerfile

```diff
@@ -21,7 +21,9 @@ COPY templates/storageclass-kadalu.replica1.yaml.j2  /kadalu/templates/storagecl
 COPY templates/storageclass-kadalu.replica2.yaml.j2  /kadalu/templates/storageclass-kadalu.replica2.yaml.j2
 COPY templates/storageclass-kadalu.replica3.yaml.j2  /kadalu/templates/storageclass-kadalu.replica3.yaml.j2
 COPY templates/external-storageclass.yaml.j2  /kadalu/templates/external-storageclass.yaml.j2
-COPY lib/kadalulib.py                /kadalu/kadalulib.py
+COPY lib/kadalulib.py                    /kadalu/kadalulib.py
+COPY cli/kubectl_kadalu/storage_list.py  /kadalu/storage_list.py
```

Comment:
Getting import error even though I have storage_list and utils in operator's image?